tooling/jbang/SampleIssueTest.java

```diff
@@ -0,0 +1,177 @@
+
+	/**
+	 * Define the configuration parameter values for your use-case
+	 */
+	private Configuration createConfiguration() {
+		Configuration configuration = new Configuration();
+
+		// Use the correct JDBC url
+		configuration.setProperty( Settings.URL, "jdbc:postgresql://localhost:5432/hreact" );
+
+		// Credentials
+		configuration.setProperty( Settings.USER, "hreact");
+		configuration.setProperty( Settings.PASS, "hreact");
+
+		// Schema generation
+		configuration.setProperty( Settings.HBM2DDL_AUTO, "create" );
+
+		// Add additional entities here
+		configuration.addAnnotatedClass(MyEntity.class);
+
+		// Query logging
+		configuration.setProperty( Settings.SHOW_SQL, "true" );
+		configuration.setProperty( Settings.HIGHLIGHT_SQL, "true" );
+		return  configuration;
+	}
+
+	@Before
+	public void createSessionFactory() {
+		Configuration configuration = createConfiguration();
+		StandardServiceRegistryBuilder builder = new ReactiveServiceRegistryBuilder()
+				.applySettings( configuration.getProperties() );
+		StandardServiceRegistry registry = builder.build();
+
+		sessionFactory = configuration.buildSessionFactory( registry )
+				.unwrap( Mutiny.SessionFactory.class );
+	}
+
+	@Test
+	public void firstTest(TestContext context) {
+		Async async = context.async();
+
+		MyEntity entity = new MyEntity( "first entity", 1 );
+		sessionFactory
+				.withTransaction( (session, tx) -> session.persist( entity ) )
+				.chain( () -> sessionFactory
+						.withSession( session -> session
+								.find( MyEntity.class, entity.getId() )
+								.invoke( foundEntity -> assertThat( foundEntity.getName() ).isEqualTo( entity.getName() ) ) ) )
+				.subscribe().with( res -> async.complete(), throwable -> context.fail( throwable )
+		);
+	}
+
+	@After
+	public void closeFactory() {
+		if ( sessionFactory != null ) {
+			sessionFactory.close();
+		}
+	}
+
+	/*
+	 * Define your hibernate entity classes and relationships.
+	 *
+	 * This initial test includes a single entity but you can create joined and/or embedded entities too
+	 *
+	 * Be sure to add all annotated entity classes in the createConfiguration() method above
+	 *    example: configuration.addAnnotatedClass(MyOtherEntity.class);
+	 */
+	@Entity(name = "MyEntity")
+	public static class MyEntity  {
+		@Id
+		public Integer id;
+
+		public String name;
+
+		public MyEntity() {}
+
+		public MyEntity(String name, Integer id) {
+			this.name = name;
+			this.id = id;
+		}
+
+		public Integer getId() {
+			return id;
+		}
+
+		public String getName() {
+			return name;
+		}
+
+		@Override
+		public String toString() {
+			return "MyEntity"
+					+ "\n\t id = " + id
+					+ "\n\t name = " + name;
+		}
+	}
+
+	public static void main(String[] args) {
+		Result result = JUnitCore.runClasses( SampleIssueTest.class);
+
+		for ( Failure failure : result.getFailures()) {
+			System.out.println(failure.toString());
+		}
+
+		System.out.println(" All unit tests succeeded:  " + result.wasSuccessful());
```

Comment:
This will always print "All unit tests succeeded: ", even if they didn't. Also there is a space at the beginning.

Reply:
fixed